src/renderer/shared/reducers/toastsReducer.js

```diff
@@ -1,6 +1,6 @@
 import uuid from 'uuid/v1';
 import { takeRight, filter } from 'lodash';
-import { LOCATION_CHANGE } from 'react-router-redux';
+// import { LOCATION_CHANGE } from 'react-router-redux';
```

Comment:
I can't seem to find something similar in the `connected-react-router` package - should we replicate this with a string of the emitted event? @mhuggins 
Same for the tests

Reply:
I'll take a look, I'm not sure what the right approach here is without digging into the new package's docs.